typescript/github-proxy/src/index.ts

```diff
@@ -0,0 +1,68 @@
+import { DISALLOWED_URL_MSG } from './errors.js';
+
+const GITHUB_API_ALLOWLIST = [
+  'https://raw.githubusercontent.com/hyperlane-xyz/hyperlane-registry/main',
+  'https://api.github.com/repos/hyperlane-xyz/hyperlane-registry',
+];
+
+export default {
```

Comment:
This is an unusual syntax. Is it something their docs recommended? Giving functions names helps improve debugger support. So `export default function proxy() {`

Reply:
Auto generated by Cloudflare tooling. not sure why it's way